Dockerfile

```diff
@@ -1,15 +1,44 @@
-FROM ubuntu
+FROM ubuntu:21.04 AS build
 
-COPY bin/console-headless /bin/red
-RUN chmod +x /bin/red
+RUN dpkg --add-architecture i386 && \
+  apt-get update && \
+  apt-get install -y curl
+
+RUN curl -L -o /usr/local/bin/red https://static.red-lang.org/dl/auto/linux/red-latest && \
+  chmod +x /usr/local/bin/red
+
+# WORKDIR /opt/rebol
+
+# RUN curl -L -O http://www.rebol.com/downloads/v278/rebol-core-278-4-10.tar.gz
+# RUN tar -xzf rebol-core-278-4-10.tar.gz
+# RUN chmod +x rebol-core/rebol
+# RUN cp rebol-core/rebol /usr/local/bin/rebol
+
+# WORKDIR /opt/red
+
+# RUN curl -L -O https://github.com/red/red/archive/refs/tags/v0.6.4.tar.gz
```

Comment:
0.6.4 release is really old. It's best to get latest `master`: https://github.com/red/red/archive/refs/heads/master.zip

Reply:
perhaps I'd better click "Start a review". :-)

Reply:
:D